source/products.html

```diff
@@ -1,23 +1,33 @@
 {% if categories.active != blank or artists.active != blank %}

```

Comment:
taking a closer look at all of this -- there's some more changes that are needed here. since this container you're working in was meant to be strictly for category/artist navigation, it only displays if there's artists or categories. so if none exist, this title will be missing entirely on the search results. 

since we're now putting the actual page title inside `nav-title` then i think that should be an h1 element and we can remove the hidden h1 that's further down the page.

Reply:
Sweet, moved it. Also had to apply a `text-transform` on products sass to avoid uppercase that is typically applied to headings. In this case it made sense to have it not transformed.